workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -33,6 +33,7 @@
 # Use floor here because multiprocessing raises an exception if this isn't an int.
 MULTIPROCESSING_WORKER_COUNT = max(1, math.floor(multiprocessing.cpu_count()/2) - 1)
 MULTIPROCESSING_CHUNK_SIZE = 2000
+INDEX_DASK_START = 20000
```

Comment:
I thought you said you were gonna do this at 200k?

Reply:
hmm yeah good catch